### ISSUE TITLE
Create FHIR NLDBI entity schema and initial utterance dataset 

### ISSUE THREAD

**Issue by allisterb** [allisterb/Micah#1]
- [ ] Create a training set of least 50 utterances of query statements and entities.
- [ ] Create a spreadsheet showing initial accuracy of NLU model based on training dataset.
